### PR TITLE
ecip-1061: move aztlan to last call

### LINKED ISSUE #177: Core Devs Call: ECIP-1061 Aztlán Upgrade
Ref [ECIP-1061](https://github.com/ethereumclassic/ECIPs/blob/master/_specs/ecip-1061.md) #81 #157 #176 

### ETC Core Devs Call - Aztlán Finalization

When: **Wednesday, November 27, 2019, 1pm UTC**, 60 minutes max.

Where: **Ethereum Classic Discord** https://discord.gg/dwxb6nf `#ecips` channel. Will use/create a voice channel _ad hoc._

### Agenda

- Quick client teams check-in
  - Parity Ethereum / Parity Tech
  - Geth Classic / ChainSafe, ETCLabs Core
  - Multi-Geth / Multi-Geth, ETCLabs Core 
  - Hyperledger Besu / ChainSafe, PegaSys
- Aztlán (ECIP-1061) needs to be either **accepted** or _updated_ (or _rejected_)
  - discuss included EIPs
  - discuss a timeline for the protocol upgrade
    - Mordor Classic and Kotti Classic testnet (February?)
    - Ethereum Classic mainnet (March?)
- anything else related to Aztlán

Please comment to add items to the agenda

### PR DESCRIPTION
**Pull request by soc1c**
closes #177

### CODE REVIEW

**Comment by bobsummerwill** [comment]
It would probably make sense, I agree, @sorpaas, to withdraw ECIP-1061 and replace it with ECIP-1072.

**Comment by soc1c** [comment]
1061 is the aztlan ecip and I'm updating it to reflect changes. that's how we always did it before you decided it's better to have multiple competing specs for the very same thing without even having a call. you are free to update your proposals. I'm updating mine and I will not get into a fight on that, so if you don't have to comment on the content of the spec, I will not spend any more time on this discussion.

File: _specs/ecip-1061.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 ### Simple Summary
 
 Enable the outstanding Ethereum Foundation _Istanbul_ network protocol upgrades on the Ethereum
-Classic network in a hard-fork code-named _Aztlán_ to enable maximum compatibility across these networks.
+Classic network without any gas-cost assumptions in a hard-fork code-named _Aztlán_ to enable 
+maximum compatibility across these networks. 
+
+The _Yingchun_ Edition refers to the preferred variant of _Aztlán_ without any repricing of state-trie 
+dependent opcodes as previously proposed in ECIP-1072.
```
Comment (YazzyYaz) [inline _specs/ecip-1061.md]: I would add a github link to the ECIP-1072 in question, but other than that, this LGTM

**Comment by zmitton** [comment]
I'm not finding it particularly easy to follow all this.

@sorpaas do you have any technical objections to the modifications made in 1061 that are different than what was agreed in the meeting or what was implied when agreeing to 1072?

**Comment by sorpaas** [comment]
@zmitton It's procedural issues rather than technical objections. You can:

* Listen to the meeting recording yourself to confirm that in the end we decided to move forward 1072.
* Read https://github.com/ethereumclassic/ECIPs/pull/199#pullrequestreview-323755348 and #176 to understand the objection, and why we decided to go with the "option" ECIP model due to Afri rushing to merge a change meant for the "just-one Aztlan".
* Read below to know that Afri has since refused to discussion and maliciously dismiss any objections. https://github.com/ethereumclassic/ECIPs/pull/199#event-2837816513

**Pull request merged**